rack-box/Docker-Hub-README.md

```diff
@@ -17,7 +17,7 @@ If you do see these resource settings, make the following changes:
 Now you are ready to start your RACK box.  Type the following command to run your RACK box on your computer:
 
 ```shell
-docker run --detach -p 3030:3030 -p 8050:8050 -p 8080:80 -p 12050-12091:12050-12091 gehighassurance/rack-box:v11
+docker run --detach -p 3030:3030 -p 8050:8050 -p8050:8050 -p 8080:80 -p 12050-12091:12050-12091 gehighassurance/rack-box:v11
```

Comment:
The -p8050:8050 is redundant because the previous option is -p 8050:8050.

Reply:
Ha!  Looks like we both fixed it and I didn't pay enough attention when merging my commit with yours.  I reverted this change, but this command line is getting unwieldy anyhow, so I've updated this PR to add `./cli/docker_start.sh`.  I left the original instructions as well since I don't know about the Windows situation, but the script should help for Unix or MacOS users, and be easier for us to avoid additional complexity for the user.